packages/contracts/test/UnirepApp.test.ts

```diff
@@ -71,7 +151,7 @@ describe('Unirep App', function () {
             const content = 'testing'
             expect(app.post(publicSignals, proof, content))
                 .to.emit(app, 'Post')
-                .withArgs(publicSignals[0], 0, content)
+                .withArgs(publicSignals[0], 0, 1, content)
```

Comment:
This is in the scope of what @FoodChain1028 is working on. So just pinging for visibility.